app/models/search/work_search_form.rb

```diff
@@ -176,16 +176,16 @@ def search_results
   ###############
 
   SORT_OPTIONS = [
-    ['Best Match', '_score'],
-    ['Author', 'authors_to_sort_on'],
-    ['Title', 'title_to_sort_on'],
-    ['Date Posted', 'created_at'],
-    ['Date Updated', 'revised_at'],
-    ['Word Count', 'word_count'],
-    ['Hits', 'hits'],
-    ['Kudos', 'kudos_count'],
-    ['Comments', 'comments_count'],
-    ['Bookmarks', 'bookmarks_count']
+    %w[Best Match _score],
+    %w[Creator authors_to_sort_on],
```

Comment:
Should authors_to_sort_on not be creators_to_sort_on to match this change?

Additionally should it not be singular as to only be sorted by a single creator

Reply:
We could, but this becomes a bigger change when that parameter name changes. 
As long as we think that's still within the scope of this ticket, I'm willing to do it.